packages/thunderbird-bin/thunderbird-bin.pacscript

```diff
@@ -0,0 +1,31 @@
+pkgname="thunderbird-bin"
+gives="thunderbird"
+pkgver="131.0.1"
+repology=("project: ${gives}")
+source=("https://archive.mozilla.org/pub/${gives}/releases/${pkgver}/linux-x86_64/en-US/${gives}-${pkgver}.tar.bz2")
+url="https://www.thunderbird.net/en-US/"
+pkgdesc="Thunderbird is a free email application that's easy to set up and customize and it's loaded with great features!"
+maintainer=("James Ed Randson <jimedrand@disroot.org>")
+arch=('amd64')
+sha256sums=("7c6a35e4ac8c0496c1c80fa2f59ff5195fcecf1cbfe1279fda863be22d7e12ce")
+
+package() {
+  cd "${srcdir}"
+  mkdir -p "${pkgdir}/lib/thunderbird"
```

Comment:
/lib is just a symlink to /usr/lib on modern systems, so use that instead

Reply:
Alright, I'll fix it that.

Reply:
Done @vigress8 

Now feel free to merge it but do not close #6582 firsts because I'll add the ESR builds after this usual editions has been merged.